File-Formats/Presentation/FAQ.md

```diff
@@ -46,3 +46,56 @@ documentation: UG
    
    You can improve the quality of converted images by specifying the image resolution. Refer – [Converting PowerPoint presentation to Images](/file-formats/presentation/getting-started#converting-powerpoint-presentation-to-images)
```

Comment:
Improve the quality of converted images by specifying the image resolution. Refer to [Converting PowerPoint presentation to Images](/file-formats/presentation/getting-started#converting-powerpoint-presentation-to-images).

Reply:
It's already available in UG